### DISCUSSION TITLE
docker-api-1 & docker-worker-1   failed to start

### DISCUSSION THREAD

**Discussion by oulongping** [langgenius/dify#11923] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

env: macOS、Docker Desktop

when I run the command below:
```shell
docker compose up -d
```
- then
docker-api-1 & docker-worker-1   failed to start

- error info:
<img width="1009" alt="截屏2024-12-21 13 20 00" src="https://github.com/user-attachments/assets/ce5a09f5-35bf-4b60-a5d5-075df895d726" />

what shoud I do?   thanks

**Comment by KCCADA** [comment]
同上问题
